front/src/common/Map/Buttons/ButtonMapSettings.tsx

```diff
@@ -1,18 +1,28 @@
 import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { GoStack } from 'react-icons/go';
+import Tipped from '../../../applications/editor/components/Tipped';
 
 type ButtonMapSettingsProps = {
   toggleMapSettings: () => void;
 };
 
-const ButtonMapSettings = ({ toggleMapSettings }: ButtonMapSettingsProps) => (
-  <button
-    type="button"
-    className="btn-rounded btn-rounded-white btn-map-settings"
-    onClick={toggleMapSettings}
-  >
-    <span className="sr-only">Settings</span>
-    <i className="icons-admin" />
-  </button>
-);
+const ButtonMapSettings = ({ toggleMapSettings }: ButtonMapSettingsProps) => {
+  const { t } = useTranslation('translation');
+  return (
+    <Tipped mode="left">
+      <button
+        type="button"
+        className="btn-rounded btn-rounded-white btn-map-settings"
+        onClick={toggleMapSettings}
+      >
+        <span className="sr-only">Settings</span>
+        <GoStack />
+      </button>
+      <span>{t('Editor.nav.select-infra')}</span>
```

Comment:
You forgot to change the key here ;)
```suggestion
      <span>{t('Editor.nav.toggle-layers')}</span>
```